src/SideFilter.jsx

```diff
@@ -0,0 +1,116 @@
+import * as React from "react";
+import {fetchCatalogueData} from './api';
+import {useEffect, useState} from "react";
+import {
+    Autocomplete,
+    Box,
+    Button,
+    Collapse,
+    Divider,
+    FormControl,
+    InputLabel,
+    List,
+    ListItem,
+    ListItemText,
+    Select,
+    TextField
+} from "@mui/material";
+import { Add, Remove } from "@mui/icons-material";
+
+
+
+const filters = [
+    {
+        label: "Title",
+        options: ["sen test study", "newTest"]
+    },
+    {
+        label: "Acronym",
+        options: ["newTest", "newTestA"]
+    }
+];
+
+export const SideFilter = () => {
+    const [openFilters, setOpenFilters] = React.useState({});
+    const [filterValues, setFilterValues] = React.useState({});
+
+    const handleToggleFilter = (filterLabel) => {
+        setOpenFilters((prevOpenFilters) => ({
+            ...prevOpenFilters,
+            [filterLabel]: !prevOpenFilters[filterLabel]
+        }));
+    };
+
+    const handleAutocompleteChange = (event, value, filterLabel) => {
+        setFilterValues((prevFilterValues) => ({
+            ...prevFilterValues,
+            [filterLabel]: value
+        }));
+    };
+
+    const handleResetAll = () => {
+        setOpenFilters({});
+        setFilterValues({});
+    };
+
+
+    return (
+        <Box sx={{ bgcolor: "#F5F5F5", width: "212px", p: "24px" }}>
+            <Box
+                display="flex"
+                justifyContent="space-between"
+                alignItems="center"
+                mb={2}
+            >
+                <Box fontSize="h6.fontSize" fontWeight="bold">
+                    Filters
+                </Box>
+                <Button variant="text" color="primary" onClick={handleResetAll}>
+                    Reset All
+                </Button>
+            </Box>
+            <List>
```

Comment:
I beieve we should use [the auto-complete](https://mui.com/material-ui/react-autocomplete) control here. Start simple, only allowing a single selection, and we'll work our way up.